src/cli/main.rs

```diff
@@ -341,12 +356,28 @@ fn format(opt: opt::Opt) -> Result<i32> {
                     let tx = tx.clone();
                     let opt = opt.clone();
 
+                    let should_skip_format = match &opt.stdin_filepath {
+                        Some(filepath) => {
+                            let ignore = get_ignore(filepath.parent().unwrap()).unwrap();
```

Comment:
Is it possible for this to ever fail when unwrapping here?
If you know that it shouldn't ever fail unless something is very wrong, then could you use `.expect()` with some sort of error message?

That should only happen if the error here is fatal though, and we want to panic and exit.
If its possible for this to sometimes fail, we should instead handle the error appropriately to allow stylua to continue working with the rest of the files.

Same comment on the other calls to `.unwrap()`

Reply:
Yep, these calls shouldn't really fail. Addressed.